lib/tokenizers.js

```diff
@@ -37,19 +37,25 @@ bedrock.events.on('bedrock-mongodb.ready', async () => {
   }]);
 });
 
-export async function get({id} = {}) {
+export async function get({id, explain = false} = {}) {
   if(!(id && typeof id === 'string')) {
     throw new TypeError('"id" must be a string.');
   }
 
+  const query = {'tokenizer.id': id};
+  const projection = {_id: 0};
+  const collection = database.collections['tokenizer-tokenizer'];
+
+  if(explain) {
+    const cursor = await collection.find(query, projection);
+    return cursor.explain('executionStats');
+  }
+
   // return cached tokenizer if available
   if(CACHED_TOKENIZER && CACHED_TOKENIZER.id === id) {
```

Comment:
Should this be a fast path and avoid the query/projection/collection initialization if possible?  If so, could add a `!explain` conditional and move the var initialization and explain block below again.

Reply:
My thought is that we do not want to add any explain code to this if statement. The cashing is unrelated to the explain and we should not interfere with it's if statement. If it is preferred to use `!explain` here instead and move all the other code below, that is fine.

Reply:
I was assuming explain=false is the general use case here?  I was trying to optimize the cached tokenizer path.  In that case, you wouldn't need query/projection/collection to be initialized.  The change here has those always being initialized.  Using `if(!explain && CACHED_....) {...}` first would be faster.  If this is over optimizing, then it's not a big deal.

Reply:
Alright I went ahead and made your change here.